delphi-frontend/src/test/resources/au/com/integradev/delphi/symbol/helpers/WeakAliasHelper.pas

```diff
@@ -0,0 +1,32 @@
+unit Simple;
+
+interface
+
+type
+  TFoo = record
+    procedure Bar;
+  end;
+
+  TFooHelper = record helper for TFoo

```

Comment:
I think you forgot to make this a weak alias 😄 I checked and this test already passes on master.

Updating this file to match the below does fail on master and pass in this branch, though, so the main code change is all good.
```delphi
type
  TFoo = record
    procedure Bar;
  end;

  TFooAlias = TFoo;

  TFooHelper = record helper for TFooAlias
    procedure Bar;
  end;
```

Reply:
Whoops, I'm a dork.
Fixing that...